composer.json

```diff
@@ -31,7 +31,8 @@
     "psr/http-message": "^1.0",
     "symfony/console": "^5.1",
     "symfony/yaml": "^5.1",
-    "twig/twig": "^3.0"
+    "twig/twig": "^3.0",
+    "webmozart/assert": "1.11.0"
```

Comment:
indeed we can explicitly declare this in composer.json but so far it was working alright bc the lib is in composer.lock required by another package, but in composer.lock it's 1.9.1, so you're missing the .lock changes in the PR (see failed pipeline)

Reply:
oh.. yes.. done.